README.md

```diff
@@ -1,20 +1,151 @@
-# Data Structures And Algorithms
+<h1 align="center"> Data Structures and Algorithms 📚💻 </h1>
 
-A collection of data structures and algorithms implemented in Python.
-Inspired from the `Design and Analysis of Algorithms` course from `3rd year, B Tech, CSE, Amrita Vishwa Vidyapeetham, Coimbatore` taught by `Dr. Vidya Balasubramanian` ma'am.
+<div align="center" style="display:flex; align-items: center; justify: center; text-decoration: none ">
+  <a href="https://github.com/Ashrockzzz2003/Data_Structures_and_Algorithms/blob/main/LICENSE" target="_blank" rel="noreferrer">
+    <img align='center' src="https://img.shields.io/badge/LICENSE-GPL-green"/>
+  </a>
+  <a href="https://github.com/Ashrockzzz2003/Data_Structures_and_Algorithms" target="_blank" rel="noreferrer">
+    <img align='center' src="https://img.shields.io/github/created-at/Ashrockzzz2003/Data_Structures_and_Algorithms"/>
+  </a>  
+</div>
 
-<!--
-## Contributions required on
+Welcome to the **Data Structures and Algorithms** repository—a comprehensive collection of foundational data structures and algorithms implemented in **Python**, **C++**, **Go**, and **Java**. This repository serves as a resource hub for **computer science students** and **enthusiasts** eager to deepen their understanding of algorithmic design.
```

Comment:
I want the old description to be there.
Don't want this one. Feels very AI generated.